addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -565,12 +565,35 @@ QUnit.module("Fields", (hooks) => {
         await click(target, ".o_field_property_selection_option:nth-child(2) .fa-trash-o");
         options = popover.querySelectorAll(".o_field_property_selection_option");
         assert.strictEqual(options.length, 3, "Should have removed the second option");
-        const optionValues = [...options].map((option) => option.querySelector("input").value);
+        let optionValues = [...options].map((option) => option.querySelector("input").value);
         assert.deepEqual(
             optionValues,
             ["A", "C", "New option"],
             "Should have removed the second option"
         );
+
+        // focus should be in option C
```

Comment:
Would add a small explanation before this.

Something like:
"Test that pressing 'Enter' inserts a new option after the one currently focused (and not last)."